Models/JsonStat/SSBAgesInNorway.cs

```diff
@@ -0,0 +1,31 @@
+using System.Text.Json.Serialization;
+
+namespace Backend.Models.JsonStat
+{
+    // Remove comments when merge-ready
```

Comment:
Just another reminder.

Reply:
In JetBrains you can add `TODO: ` to have it show up in the TODO pane (though the comments feel uncomfortable in `master`).